docs/current_docs/cookbook/snippets/custom-spans/python/main.py

```diff
@@ -0,0 +1,64 @@
+import asyncio
+from dagger import dag, function, object_type
+from opentelemetry import trace
```

Comment:
@helderco this code fails with multiple errors, see https://dagger.cloud/dagger/traces/187eadadb7e64c78f41550a01b305272. Can you please help to fix it?